src/messages.js

```diff
@@ -68,4 +68,17 @@ export default {
   'vocab.organization.local.collectionName': 'Organizations',
   'field.organizations_common.historyNote.name': 'Organization note',
   'field.organizations_common.contactName.name': 'Contact/chair',
+
+  // Distribution
+  'record.objectexit.name': 'Distribution',
+  'record.objectexit.collectionName': 'Distributions',
+  'field.objectexit_common.exitNumber.name': 'Distribution number',
+  'panel.objectexit.info': 'Distribution Information',
+  'field.objectexit_common.exitMethod.name': 'Distribution method',
+  'field.objectexit_common.exitReason.name': 'Distribution reason',
+  'field.objectexit_common.exitNote.name': 'Distribution note',
+  'field.objectexit_common.currentOwner.name': 'Authorized by',
+  'field.objectexit_common.depositor.name': 'Organization',
+  'field.objectexit_common.count.name': 'Count',
```

Comment:
Don't need count message, it's already defined in the naturalhistory objectexit extension.